create.js

```diff
@@ -190,22 +205,58 @@ exports.write_3 = function(name, variant, dev) {
 
 // 'Interdisciplinary' Folder ---------------
 
-exports.write_3("Business", "Risk Assessment");
-exports.write_3("Business", "CAT");
-exports.write_3("Business", "Business Use Security");
-exports.write_3("Business", "Government Use Security");
-exports.write_3("Business", "Personal Use Security");
-exports.write_3("Healthcare Management", "Risk Management");
-exports.write_3("Healthcare Management", "HIPPA");
-exports.write_3("Healthcare Management", "Hipaa");
-exports.write_3("Mitigating Risk", "Value Modeling");
-exports.write_3("Security Training for Election Judges", "Ensuring Pollbook Security");
-exports.write_3("Security Training for Election Judges", "Ensuring Provisional Voting Security");
-exports.write_3("Security Training for Election Judges", "Operating the Scanning Unit");
-exports.write_3("Security Training for Election Judges", "Ensuring Pollbook Security Anne Arundel");
-exports.write_3("Security Training for Election Judges", "Ensuring Provisional Voting Security Anne Arundel");
-exports.write_3("Security Training for Election Judges", "Operating the Scanning Unit Anne Arundel");
-exports.write_3("Security Training for Election Judges", "Chief Judge Security Anne Arundel");
-exports.write_3("Security Training for Election Judges", "Ensuring Ballot-Marking Devices Security Anne Arundel");
-exports.write_3("Security Training for Election Judges", "Ensuring Voting Booth Security Anne Arundel");
-exports.write_3("Security Training for Election Judges", "Same Day Registration");
\ No newline at end of file
+// exports.write_3("Business", "Risk Assessment");
```

Comment:
We may need to ask @pzalep1, are there any modules that must always be built?

Reply:
Only time we would ever need to do an update like that would be if we make an update to the template. Kind of what happened last month when there was a link in the script that was returning a 500 and that script tag was in the template.